moto/swf/models/domain.py

```diff
@@ -1,18 +1,19 @@
 from __future__ import unicode_literals
 from collections import defaultdict
 
-from moto.core import BaseModel
+from moto.core import ACCOUNT_ID, BaseModel
 from ..exceptions import (
     SWFUnknownResourceFault,
     SWFWorkflowExecutionAlreadyStartedFault,
 )
 
 
 class Domain(BaseModel):
-    def __init__(self, name, retention, description=None):
+    def __init__(self, name, retention, description=None, swf_backend=None):
```

Comment:
I think we should pass in the region directly, instead of passing in the entire backend-object

Reply:
Done. The original implementation followed what I saw in EMR, but this works too. 😃